tests/LOOP_ROOMS/test_ROOMS126.py

```diff
@@ -0,0 +1,156 @@
+        self.email = Email(self)
+        self.browser = Browser(self)
+
+        self.connect_to_network()
+
+        self.user = self.UTILS.general.get_config_variable("gmail_1_user", "common")
+        self.emailadd = self.UTILS.general.get_config_variable("gmail_1_email", "common")
+        self.passwd = self.UTILS.general.get_config_variable("gmail_1_pass", "common")
+        self.hot_user = self.UTILS.general.get_config_variable("hotmail_1_user", "common")
+        self.hot_emailadd = self.UTILS.general.get_config_variable("hotmail_1_email", "common")
+        self.hot_passwd = self.UTILS.general.get_config_variable("hotmail_1_pass", "common")
+        self.email.launch()
+        self.email.setupAccount(self.user, self.emailadd, self.passwd)
+        self.apps.kill_all()
+        time.sleep(2)
+
+        self.loop.initial_test_checks()
+
+        self.apps.kill_all()
+        time.sleep(2)
+        self.test_room = "Room{}".format(time.time())
+        self.new_name = self.test_room + "--New"
+        self.fxa_user = self.UTILS.general.get_config_variable("fxa_user", "common")
+        self.fxa_pass = self.UTILS.general.get_config_variable("fxa_pass", "common")
+        self.test_contact = MockContact(email={'type': 'Personal', 'value': self.fxa_user})
+        self.UTILS.general.insertContact(self.test_contact)
+
+    def tearDown(self):
+        self.UTILS.reporting.reportResults()
+        GaiaTestCase.tearDown(self)
+
+    def test_run(self):
+        # First, login
+        self.loop.launch()
+        result = self.loop.wizard_or_login()
+
+        if result:
+            self.loop.phone_login_auto()
+            self.loop.allow_permission_phone_login()
+            self.UTILS.element.waitForElements(DOM.Loop.app_header, "Loop main view")
+
+        self.UTILS.iframe.switchToFrame(*DOM.Loop.frame_locator)
+        self.UTILS.reporting.debug("Creating room with name: {}".format(self.test_room))
+        self.loop.create_room(self.test_room)
+        time.sleep(5)
+        self.loop.edit_room(self.new_name)
+
+        # Check the new room name in the details view
+        time.sleep(5)
+        self.wait_for_element_displayed(*DOM.Loop.room_name)
+        name = self.marionette.find_element(*DOM.Loop.room_name)
+        self.UTILS.test.test(name.text == self.new_name, "New name for room is [{}]   Expected: [{}]".
+                             format(name.text, self.new_name))
+
+        # Share the room with the user we have just created
+        self.loop.share_room(self.test_contact, by_sms=False)
+
+        # Now logout and log in with another user
+        self.marionette.find_element(*DOM.Loop.room_back_btn).tap()
+        self.loop.logout()
+        time.sleep(3)
+        self.loop.firefox_login(self.fxa_user, self.fxa_pass)
+        self.UTILS.iframe.switchToFrame(*DOM.Loop.frame_locator)
+        self.loop.tap_on_firefox_login_button()
+
+        self.email.launch()
+        self.email.remove_accounts_and_restart()
```

Comment:
There's no need of removing the previous accounts. setupAccount() can be used with any worries. :D